### PR TITLE
Fix include guards and copyright statements

### PR DESCRIPTION
**Pull request by messmerd**
This PR corrects some minor issues with header files under `include/`
- Added namespace prefixes (`LMMS_` or `LMMS_GUI_`) to include guards of header files in the `include/` directory. This fixes a name conflict between the include guards of `include/Engine.h` and `plugins/ZynAddSubFx/zynaddsubfx/src/Nio/Engine.h` and possibly others. It will also prevent any future conflicts.
- The include guards of `Clip.h` and `ClipView.h` were still using their old "track content object" names from before the reorganization. This has been corrected. ([#5592](https://github.com/LMMS/lmms/issues/5592))
- Added missing copyright statement to `MidiCCRackView.h`.
- Added missing copyright statements to `CustomTextKnob.h` and `CustomTextKnob.cpp` according to author's comment in [#5321](https://github.com/LMMS/lmms/pull/5321#issuecomment-1371783964).
- Makes the number of newlines between copyright statement and include guard, between include guard and includes, and between closing include guard and EOF consistent for all headers

### CODE REVIEW

**Review by Veratil — COMMENTED** [review]
LGTM

My only note is a few of the `#endif`'s lost their `// GUARD_H` comment. We don't really need them since they're EOF, but it doesn't hurt to have I think?

**Comment by messmerd** [comment]
> My only note is a few of the `#endif`'s lost their `// GUARD_H` comment. We don't really need them since they're EOF, but it doesn't hurt to have I think?

There were only a few instances of those, so I just removed them for consistency with the rest of the headers.

**Comment by messmerd** [comment]
Actually, I'll go ahead and add them to all the headers. It would be better to have them, and now's the best time to do it.

**Comment by PhysSong** [comment]
Generally looks good.
I'm not sure if it's necessary, but I think it's a good chance to rename some include guards, ex. `LMMS_AUDIO_DEVICE_H` to `LMMS_AUDIO_DEVICE_H`. Historically, some header files used names in snake_case, but they were renamed to PascalCase.

File: include/aeffectx.h [hunk 1]
```diff
@@ -22,9 +22,8 @@
  *
  */
 
-
-#ifndef AEFFECTX_H
-#define AEFFECTX_H
+#ifndef LMMS_AEFFECTX_H
```
Comment (DomClark) [inline include/aeffectx.h:25]: This one isn't in the LMMS namespace.

**Review by DomClark — APPROVED** [review]

**Comment by Veratil** [comment]
Since @PhysSong has soft approval with comment, and @DomClark has approval. I'm merging this now.

**Pull request merged**